internal/js/modules/k6/browser/tests/network_manager_test.go

```diff
@@ -19,6 +19,9 @@ import (
 func TestURLSkipRequest(t *testing.T) {
 	t.Parallel()
 
+	// Today's date: 2025-09-09
```

Comment:
```suggestion
	// TODO: link the issue
```

Reply:
It would be good to open an issue and link here

Reply:
```suggestion
	// t.Skip("Temporary skip while we work out why test is failing with the latest version of chromium; TODO: link here")
```
Or directly something like this

Reply:
```suggestion
t.Skip("Temporary skip while we work out why test is failing with the latest version of chromium; TODO: https://github.com/grafana/k6/issues/5162")
```

Reply:
Changes done in 8510401ae38071359376c3af2750c56eb998f9e7 and 7bf320230d327a5c0ad356bd8b63cab58a9e9c93